src/Interpreters/Cache/Metadata.h

```diff
@@ -22,7 +22,7 @@ struct FileSegmentMetadata : private boost::noncopyable
 
     size_t size() const;
 
-    bool valid() const { return !removal_candidate.load(); }
+    bool evicting() const { return !removal_candidate.load(); }
```

Comment:
Valid and evicting seem to have opposite meaning, but both are `!removal_candidate`.
Isn't this a bug?

Reply:
ops, thanks